cluster-autoscaler/Godeps/Godeps.json

```diff
@@ -71,33 +71,33 @@
 		},
 		{
 			"ImportPath": "github.com/Azure/go-autorest/autorest",
-			"Comment": "v10.6.2",
-			"Rev": "1ff28809256a84bb6966640ff3d0371af82ccba4"
+			"Comment": "v10.12.0",
```

Comment:
This version doesn't match the one in kubernetes/kubernetes and the package isn't included in _override: https://github.com/kubernetes/kubernetes/blob/master/Godeps/Godeps.json#L75

Reply:
hmm, yep. Kubernetes doesn't upgrade to this version yet. And the fix should also be cherry-picked to old versions. Let me add it in _override instead.